packages/client/merge/README.md

```diff
@@ -85,14 +86,18 @@ In case you want to host `engine_*` without auth (not recommended, only for debu
 
 This will by default try reaching out `ethereumjs` over the endpoint `8551`. (You may customize this in conjuction with `ethereumjs`, see lodestar cli help via `--help`).
 
-You may provide `--weakSubjectivityServerUrl` (with a synced `sepolia` beacon node endpoint as arg value) and `--weakSubjectivitySyncLatest` to start directly off the head on the `sepolia` beacon chain, possibly triggering (backfill) beacon sync on ethereumjs.
+You may provide `--checkpointSyncUrl` (with a synced `sepolia` beacon node endpoint as arg value) to start directly off the head/provided checkpoint on the `sepolia` beacon chain, possibly triggering (backfill) beacon sync on ethereumjs.
 
 #### (Optional) Validator
 
-To run the validator, you will need to add the following arg to the above beacon run command to expose api endpoints: `--api.rest.enabled`. The validator will connect to the beacon node via these apis (Refer to `lodestar` cli help to modify the default endpoints for both beacon and validator)
+Once you start beacon node instructed as above, it will expose some default api endpoints. The validator started as instructed below will connect to the beacon node via these apis (Refer to `lodestar` cli help to modify the default endpoints for both beacon and validator)
```

Comment:
you mean beacon and validator belonging to different CL clients? yes, its generally possible because the apis are spec-ed and standardized. 
though one might run into some issues (which should be easily rectifiable)